sar_pipeline/aws/preparation/orbits.py

```diff
@@ -108,7 +108,7 @@ def download_orbits(
     logger.info(f"Starting EOF download from {source}...")
 
     # The logic in eof.download.main() tries CDSE first by default. set force_asf by source
-    return eof.download.main(
+    ORBIT_PATHS = eof.download.main(
```

Comment:
```suggestion
    orbit_paths = eof.download.main(
```
Not sure this needs to be in caps indicating a constant.